components/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -84,7 +86,12 @@ class D2LFilterDropdown extends mixinBehaviors([D2L.PolymerBehaviors.FilterDropd
 			disableOpenerTextVariation: {
 				type: Boolean,
 				value: false
-			}
+			},
+			disabled: {
+				type: Boolean,
+				reflectToAttribute: true,
```

Comment:
You shouldn't need attribute reflection on either of these, since there's no CSS attribute selectors that depend on the value.